playbooks/ovirt-host-update.yml

```diff
@@ -0,0 +1,17 @@
+- hosts: all
```

Comment:
We have moved the playbooks to the ovirt-engine project. Can you please move this one as well? You can create it as part of the patch: https://gerrit.ovirt.org/#/c/84313/2/

You can see the structure here: https://gerrit.ovirt.org/#/c/83870/